<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AcoParametersDialog</class>
 <widget class="QDialog" name="AcoParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>236</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CoreAssigner Parameters</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="minCoresLabel">
     <property name="text">
      <string>minimum number of cores</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSpinBox" name="minCoresBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="maxCoresLabel">
     <property name="text">
      <string>maximum number of cores</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="maxCoresBox">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="maxWeightPerCoreLabel">
     <property name="text">
      <string>maximum weight per core</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="dataStreamWeightLabel">
     <property name="text">
      <string>datastream connector weight</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="controlStreamWeightLabel">
     <property name="text">
      <string>controlstream connector weight</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="maxWeightPerCoreBox">
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="dataStreamWeightBox">
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="controlStreamWeightBox">
     <property name="maximum">
      <number>2147483647</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Abort|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AcoParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AcoParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
